src/test/java/com/github/jhonnymertz/wkhtmltopdf/wrapper/integration/PdfIntegrationTests.java

```diff
@@ -63,6 +66,25 @@ public void testPdfFromStringTo() throws Exception {
         Assert.assertThat("document should contain the creditorName", pdfText, containsString("Müller"));
     }
 
+    @Test
+    public void testMultipleObjectsWithOptions() throws Exception {
```

Comment:
In the issue you created you added: `The current wrapper always puts the table of contents first, removing the ability to specify where in the document it should be placed.
Additionally, the amendments and foreword shouldn't be included in the table of contents, however, the wrapper currently doesn't support per page parameters so specifying --exclude-from-outline for these pages only isn't possible.`,

do you think we can add unit tests that specifically validates the ability to put the table of contents before or after a section in the document?

Regarding the second point in the comment about `--exclude-from-outline`, does this PR contemplate such implementation? Otherwise I'll create an issue to capture this desire and attempt to address in the future.

Reply:
Also, could you please add more tests with different combinations of the commands stressing what your implementation is covering now? I'll use them later as a reference to update the documentation later and highlight the new capabilities of the tool.

Reply:
Yes, agreed, I will look into adding the following tests:

- verify toc position (add after a cover page for example)
- verify that per page options are working
    - Set a different footer on two seperate pages and verify they appear in the correct order
    - Setup pdf with: toc, page 1, page 2. set `--exclude-from-outline` on page 1 and assert that only page 2 shows in the toc.
    - Others to cover more usages and variations of the above (using other object [toc,cover,page] specific options)

Lastly, In regards to: ```Regarding the second point in the comment about --exclude-from-outline, does this PR contemplate such implementation? Otherwise I'll create an issue to capture this desire and attempt to address in the future.``` 

This PR covers this usage as well, no need for an additional issue (can add params to a page now and can add params to the pdf [global]). However, defining useful methods to add object (cover,toc,page) specific options to their params list could be a nice follow on.

Reply:
Perfect, the unit test scenarios look good, thank you for this! I'll make sure to capture these usages and update the documentation later to make it clear to people that these features exist and how they can be used.

I captured an [issue regarding the useful methods to add object options](https://github.com/jhonnymertz/java-wkhtmltopdf-wrapper/issues/112), we can address it separately. Thank you for the explanation.